lib/pynput/keyboard/_xorg.py

```diff
@@ -88,6 +88,7 @@ class Key(enum.Enum):
     alt_l = KeyCode._from_symbol('Alt_L')
     alt_r = KeyCode._from_symbol('Alt_R')
     alt_gr = KeyCode._from_symbol('Mode_switch')
+    level3_shift = KeyCode._from_symbol('ISO_Level3_Shift')
```

Comment:
```suggestion
    alt_gr = KeyCode._from_symbol(''ISO_Level3_Shift')
    mode_switch = KeyCode._from_symbol('Mode_switch')
```
Most xkb layouts in `/usr/share/X11/xkb/symbols` by far assign the `'ISO_Level3_Shift` to the right AltGr key. Only `digital_vndr/pc` `nec_vndr/jp` `pk` `sun_vndr/tw` and `sun_vndr/jp` use `Mode_switch`. With that mind, it makes more sense to swap the two.

PS: The alphabetical order is broken with my suggestion so don't hit the "Apply" button. Move `mode_switch` between `left` and `page_down` to preserve the alphabetical order.

Reply:
Double quote ''ISO_Level3_Shift'